core/XH.js

```diff
@@ -271,6 +271,20 @@ class XHClass {
         return this.router.navigate(...args);
     }
 
+    /** Add a routeName to the current route, preserving params */
```

Comment:
Should we move these to routerModel?  Will application code need to call them?  Trying to make XH be more about common aliases than implementation, and to keep it from getting too bloated

Reply:
Mobile apps will use these as the primary methods for navigation (far more often that `XH.navigate()`). I've moved the implementation to RouterModel and left the XH aliases.